### DISCUSSION TITLE
Dendron Support

### DISCUSSION THREAD

**Discussion by ratsey** [gsantner/markor#1738] [Ideas] [answered]
I'd love to see Markor provide support for dot hierarchy notes such as created by [Dendron](https://www.dendron.so/). 

When currently viewing the Files, the flat file system that Dendron implements lists all files together. It would be ideal if Markor could parse the dot notation and implement virtual folders for the display.

_e.g._
```
top.next.myfile.md
top.next.anotherfile.md
```

would render as

```
top
> next
> > myfile.md
> > anotherfile.md
```

Bonus points for substituting the YAML header title for the filenames

**Comment by gsantner** [comment, marked as answer]
It's the first time I hear about this and it sounds like some feature specific of Dendron. Usually at Markor we try to make things that work with most Markdown applications and don't have a kind of vendor lock-in/vendor specific.

But if you want this, please implement it and make a merge request.
